app/components/access_panels/other_listings_component.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module AccessPanels
+  class OtherListingsComponent < AccessPanels::Base
+    def render?
+      document.druid.present?
+    end
+
+    def earthworks_url
+      @earthworks_url ||= construct_earthworks_url
+    end
+
+    def purl_url
+      "#{Settings.PURL_EMBED_RESOURCE}#{document.druid}"
+    end
+
+    private
+
+    def construct_earthworks_url
+      response = Net::HTTP.get_response(URI.parse("#{purl_url}.meta_json"))
```

Comment:
Can we do this on the front end so that it doesn't block page drawing?

Reply:
Done